src/store/index.js

```diff
@@ -41,9 +43,7 @@ const store = new Vuex.Store({
      * @return {Object}
      */
     async getNodeStatus ({ state, commit, dispatch }) {
-      const client = await EpochChain({
-        url: this.state.epochUrl
-      })
+      let client = await getEpochClient(this.state.epochUrl)
```

Comment:
We should use `const` instead of `let`

Reply:
yep. will fix this. I had earlier wired this up in a different way but modified before making the commit must have forgotten to update it.